examples/components/typography/typography.css

```diff
@@ -1,3 +1,30 @@
 h1 {
+  font-family: var(--ff-primary);
   font-size: 2rem;
+  font-weight: var(--fw-primary-bold);
+}
+h2 {
+  font-family: var(--ff-primary);
+  font-size: 1.75rem;
+  font-weight: var(--fw-primary);
+}
+h3 {
+  font-family: var(--ff-primary);
+  font-size: 1.5rem;
+  font-weight: var(--fw-primary);
+}
+h4 {
+  font-family: var(--ff-secondary);
+  font-size: 1.25em;
+  font-weight: var(--fw-secondary);
+}
+h5 {
+  font-family: var(--ff-primary);
+  font-size: 1rem;
+  font-weight: var(--fw-primary);
+}
+h6 {
+  font-family: var(--ff-primary);
+  font-size: 1rem;
+  font-weight: var(--fw-primary);
```

Comment:
But this file is for the examples, this shouldn't get changed

Reply:
Yes, you are right. I was at first in the examples folder. I change it and will commit it!